### PR TITLE
Stoic21 example

### PR DESCRIPTION
**Pull request by mosheraboh**

### CODE REVIEW

File: examples/fuse_examples/imaging/classification/stoic21/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# STOIC21 Classification with Fuse 
+
+Example of severe COVID-19 classifier baseline given a Computed-Tomography (CT), age group and gender.
+More details about the challenge can be found here: https://stoic2021.grand-challenge.org/
+
+# Data
+The STOIC Dataset (see https://pubs.rsna.org/doi/10.1148/radiol.2021210384 for a full description) contains Computed Tomography scans from 10,735 patients.  For this challenge, one CT scan from each patient has been selected, and the dataset has been divided randomly into a public training set (2,000 patients), a test set (~1,000 patients)
```
Comment (alex-golts) [inline examples/fuse_examples/imaging/classification/stoic21/README.md:7]: the rest is private *taining* set? can you clarify this? it means it's training set that only people with authorization can use?

File: examples/fuse_examples/imaging/classification/mnist/runner.py [hunk 2]
```diff
@@ -258,10 +258,6 @@ def run_infer(paths: dict, infer_common_params: dict):
     lgr.info(f'infer_filename={os.path.join(paths["inference_dir"], infer_common_params["infer_filename"])}', {'color': 'magenta'})
 
     ## Data
-    transform = transforms.Compose([
```
Comment (alex-golts) [inline examples/fuse_examples/imaging/classification/mnist/runner.py:261]: it's not needed anymore for the mnist example?

File: fuse/data/utils/split.py [hunk 3]
```diff
@@ -12,97 +12,132 @@
...
+def balanced_division(df : pd.DataFrame, no_mixture_id : str, keys_to_balance: Sequence[str], nfolds : int, seed : int=1357,
+                        excluded_samples: Sequence[Hashable]=tuple(), print_flag : bool =False, debug_mode : bool=False) -> pd.DataFrame: 
+    """
+    Partition the data into folds while using no_mixture_id for which no mixture between folds should be forced.
+    and using keys_to_balance as the keys for which balancing is forced.
+    The functions creates ID level labeling which is the cross-section of all possible mixture of key columns for that id
+    it creates the folds so each fold will have about same proportions of ID level labeling while each ID will appear only in one fold
+    For example - patient with ID 1234 has 2 images , each image has a binary classification (benign / malignant) .
+    it can be that both of his images are benign or both are malignant or one is benign and the other is malignant.
+    For example
```
Comment (alex-golts) [inline fuse/data/utils/split.py:52]: remove "For example"?

File: fuse/dl/models/heads/common.py [hunk 4]
```diff
@@ -46,6 +46,31 @@ def forward(self, x):
         x = self.classifier(x)
         return x
 
+class ClassifierFCN3D(nn.Module):
```
Comment (alex-golts) [inline fuse/dl/models/heads/common.py:49]: can you give a short description? 
also to explain how it's different from Head3dClassifier

**Review by alex-golts — DISMISSED** [review]
Looks good to me! 
See few small comments inline

File: examples/fuse_examples/imaging/classification/mnist/runner.py (hunk 2 above)
Comment (mosheraboh) [reply examples/fuse_examples/imaging/classification/mnist/runner.py:261]: It is in fuseimg/datasets/mnist.py.
This is a leftover I forgot to delete.

File: fuse/utils/multiprocessing/run_multiprocessed.py [hunk 5]
```diff
@@ -232,4 +232,25 @@ def get_from_global_storage(key: str) -> Any:
     Get args copied by run_multiprocessed
     """
     global _multiprocess_global_storage
-    return _multiprocess_global_storage[key]
\ No newline at end of file
+    return _multiprocess_global_storage[key]
+
+
+def run_in_subprocess(f, timeout=600):
```
[the remark this answers is not in the document]
Comment (mosheraboh) [reply fuse/utils/multiprocessing/run_multiprocessed.py:238]: This one is a decorator that runs functions in a subprocess.
Used for tests that use GPU, to make sure the GPU is released when after the test.
I see that I've missed type annotation and some description though.

**Pull request merged**